latch_cli/services/cp/cp.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from latch_cli.services.cp.config import CPConfig, Progress
+from latch_cli.services.cp.download import download
+from latch_cli.services.cp.upload import upload
+from latch_cli.services.cp.utils import is_remote_path
+
+
+def cp(
+    src: str,
+    dest: str,
+    *,
```

Comment:
why?

Reply:
seemed cleaner to have configuration params be keyword-only

Reply:
oh okay, yea I think that makes sense.